src/combineKeySelectors.js

```diff
@@ -0,0 +1,22 @@
+function combineKeySelectors({inputSelectors = [], keySelector}) {
+  const keySelectors = inputSelectors
+    .filter(entry => entry.hasOwnProperty('keySelector'))
```

Comment:
I think it will more performant if you will filter inputSelectors by `typeof keySelector === 'function'` here.